packages/components/src/DateTimePickers/InputDateTimePicker/InputDateTimePicker.component.js

```diff
@@ -187,6 +210,46 @@ class InputDateTimePicker extends React.Component {
 		this.updateDateTime(date, time, fullString, errMsg);
 	}
 
+	onFocusInput() {
+		this.switchDropdownVisibility(true);
+	}
+
+	setContainerRef(ref) {
+		this.containerRef = ref;
+	}
+
+	setDropdownWrapperRef(ref) {
+		this.dropdownWrapperRef = ref;
+	}
+
+	mountDocumentHandler() {
+		document.addEventListener('click', this.documentHandler);
+		document.addEventListener('focus', this.documentHandler);
+	}
+
+	unmountDocumentHandler() {
+		document.removeEventListener('click', this.documentHandler);
+		document.removeEventListener('focus', this.documentHandler);
+	}
+
+	documentHandler(e) {
+		const isActionOutOfComponent = e.path.every(node => node !== this.containerRef);
```

Comment:
path is compatible for all browsers ?

Reply:
Effectively it's not !
After some reading, for info :
`event.path` was introduced by chrome and then an `event.composedPath()` has been specified and implemented by the latest Chrome and FF versions but not yet for IE and Edge.

It seems there is no working polyfill, surely because it seems not doable : the path has to be calculated at the time the event is fired, because when it's handled by handlers, the dom structure can have changed and some elements could have been removed/added.
I tried a naive `event.composedPath()` polyfill based on [this repo](https://github.com/DieterHolvoet/event-propagation-path) without success.

Finally I had to do a workaround with checking if the event bubble up in the current component by hand with a second set of handlers.